CONVENTIONS.md

```diff
@@ -63,29 +63,57 @@ Figure example:
 ```
 
 <a id="syntax-namespace"></a>
-6. **Namespace for `schema.org`.** Use `https://schema.org/`. 
-
-  Consistent representation of namespaces simplifies programmatic processing of markup. For example, even though conceptually it is clear the terms `http://schema.org/Dataset` and `https://schema.org/Dataset` (note the protocol difference) are referring to [https://schema.org/Dataset](https://schema.org/Dataset), these are programmatically treated as different entities. The [schema.org guidelines](https://schema.org/docs/faq.html#19) are somewhat ambivalent on the topic, with perhaps emphasis on `"https"`. 
-
-  The trailing slash (`/`) is also important. Without it, common RDF processing libraries such as [rdflib](https://rdflib.readthedocs.io/en/stable/) will construct a term like `"https://schema.orgDataset"`. For example:
-
-  ``` python console
-  >>> from rdflib import ConjunctiveGraph
-  >>> json = """{
-  ...    "@context": {"@vocab": "https://schema.org"},
-  ...    "@id":"demo",
-  ...    "@type":"Dataset"
-  ... } """
-  >>> g = ConjunctiveGraph().parse(data=json, format="json-ld", publicID="https://my.data/")
-  >>> for s,p,o in g:
-  ...     print(f'"{str(s)}", "{str(p)}", "{str(o)}"')
-  ...
-  "https://my.data/demo", "http://www.w3.org/1999/02/22-rdf-syntax-ns#type", "https://schema.orgDataset/"
-  ```
-  Including the trailing slash will make the literal representation of terms align with the Internet location of the term definition, and so be clearer for readers and other processors.
-
-  It is further recommended that the prefix `SO:` is used in documentation and other locations when specifically referring to `https://schema.org/`.
-  
+6. **Namespace for `schema.org`.** 
+
+We recommend that the namespace URI for the schema.org vocabulary be consistently set to the value `http://schema.org/`.
+
+ Consistent representation of namespaces simplifies programmatic processing of schema.org markup. For example, even though conceptually it is clear that the terms `http://schema.org/Dataset` and `https://schema.org/Dataset` (note the protocol difference) are referring to [http://schema.org/Dataset](https://schema.org/Dataset), these are programmatically treated as different entities. The [schema.org guidelines](https://schema.org/docs/faq.html#19) are somewhat ambivalent on the topic, but their context files maintain an `http` namespace for all terms, but that can also be resolved from `https`-based addresses.
```

Comment:
Last sentence is a little awkward. Perhaps:

The [schema.org guidelines](https://schema.org/docs/faq.html#19) are somewhat ambivalent on the topic. Their context files maintain an `http` namespace for all terms, but can also be resolved from `https`-based addresses.

Reply:
Sounds good, I added that change.